### DISCUSSION TITLE
Whats the difference between the Internal and Imported Playbooks?

### DISCUSSION THREAD

**Discussion by SimonCampbell85** [Security-Onion-Solutions/securityonion#9366] [General] [answered]
Hi There,

I have been using the Playbook app on Security Onion and I noticed that there are 3 playbooks.

- Community
- Imported
- Internal

The Community and Imported Playbooks seem straightforward, they contain Plays that are from the community, or that I have created using a Sigma rule. What is the purpose of the Internal Playbook?

Many thanks, Simon

**Comment by defensivedepth** [comment, marked as answer]
Just a default category, that's all!
